beets/dbcore/db.py

```diff
@@ -217,6 +217,18 @@ def _check_db(self, need_id=True):
         if need_id and not self.id:
             raise ValueError(u'{0} has no id'.format(type(self).__name__))
 
+    def copy(self):
+        """Create a semi-deep copy of the item. In particular, the _db object
+        is not duplicated. A simple copy.deepcopy wouldn't work due to the
+        sqlite objects in _db.
+        """
```

Comment:
I know this is extremely low-level, but sometimes I like structuring docstrings so there is a quick summary at the top followed by more detail after a blank line. This can help when navigating a big codebase for the first time. For example, I might suggest:

```
"""Create a copy of the model object.

The field values and other state is duplicated, but the new copy
remains associated with the same database as the old object.
(A simple `copy.deepcopy` will not work because it would try to
duplicate the SQLite connection.)
"""
```

Reply:
:+1: